### DISCUSSION TITLE
Restoring db.sqlite3 to a different version

### DISCUSSION THREAD

**Discussion by olliestrickland** [dani-garcia/vaultwarden#5301] [Q&A] [answered]
Can a database that originates from a 1.32.6 instance be restored to an 1.32.5 instance?

Goal - move from Docker to FreeBSD, but the latest vaultwarden package in the FreeBSD repo is 1.32.5.

If there is any chance of a problem I'll just wait until FreeBSD catches up in a few weeks.

**Comment by BlackDex** [comment, marked as answer]
There were no database changes since 1.32.1. So yes.
